src/components/ExerciseHeader.tsx

```diff
@@ -43,12 +43,12 @@ interface ExerciseHeaderProps {
 }
 
 const ExerciseHeader = ({
-  navigation,
-  closeExerciseAction,
-  currentWord,
-  numberOfWords,
-  confirmClose = true
-}: ExerciseHeaderProps): JSX.Element => {
+                          navigation,
```

Comment:
I guess this will be fixed by prettier?

Reply:
yes somehow my prettier config didn't run on format. Gonna fix with next commit